src/Services/PatientService.php

```diff
@@ -113,45 +143,19 @@ public function getFreshPid()
     public function insert($data)
     {
         $fresh_pid = $this->getFreshPid();
+        $data['pid'] = $fresh_pid;
+        $data['pubpid'] = $fresh_pid;
+        $data['date'] = date("Y-m-d H:i:s");
```

Comment:
Is it overwriting the data from parsePatientResource
FhirPatientService.php 
line 57

Reply:
yep, it's an insert and required for new patient. don't think we want client deciding new pid's.
Also, we need more validation for duplicate patients. The date i'm a little indecisive about.